src/chapters/booleans-and-conditionals/exercises.rst

```diff
@@ -0,0 +1,140 @@
+Chapter 5 Exercises
+===================
+
+Attempt these excersises to test your understanding. 
+Don't worry if you struggle while working on them. Struggling and recalling
+the material will make you remember it.
+
+| Be sure to ask about the topics you don't understand in class, you won't be the only person that needs help with that topic.
+    
+.. note::
+
+   When the term **print** is used, that means to output a value to the ``console`` 
+   using ``console.log("text or a variable name")``.  
+   
+   | Print ``"hello"`` means ``console.log("hello")``
+
+Declare and initialize variables in the table
+----------------------------------------------
+Write variables for these values.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Variable
+     - Value
+   * - engineIndicatorLight
+     - red blinking
+   * - spaceSuitsOn
+     - true
+   * - shuttleCabinReady
+     - true
+   * - crewStatus
+     - spaceSuitsOn && shuttleCabinReady
+   * - computerStatusCode
+     - 200
+   * - shuttleSpeed
+     - 15000
+
+Read the below code. What will be printed to the console?
+----------------------------------------------------------------------
+Answer using the value of ``engineIndicatorLight`` that is defined in the table above.
+
+.. code-block:: javascript
+
+   if (engineIndicatorLight === "green") {
+      console.log("engines have started");
+   } else if (engineIndicatorLight === "green blinking") {
+      console.log("engines are preparing to start");
+   } else {
+      console.log("engines are off");
+   }
+
+Write conditional expressions to satisfy the safety rules below
+----------------------------------------------------------------
+Using the variables defined from the table above.
+
```

Comment:
I would remove one of the parts and make this a two part problem so that these exercises don't get too long.